Classes/Domain/Model/Component.php

```diff
@@ -117,6 +111,9 @@ public function getFixtures(): array
                 $loader = GeneralUtility::makeInstance(YamlFileLoader::class);
                 $fixtures = $loader->load($fixtureFile) ?? [];
                 break;
+            case 'php':
+                $fixtures = require $fixtureFile;
```

Comment:
Maybe we should validate that there is an array coming? Also the other cases take care of failures and set to empty array. 

I think both could achieved with this array cast? 
```suggestion
                $fixtures = (array) require $fixtureFile;
```

Reply:
I would suggest throwing an exception if it is not an array. Silently casting a value to an array may lead to unexpected behavior, making it difficult to trace the source of the issue. Throwing an exception would set a clear expectation.

Reply:
I've added a general exception because also other fixture type can return unexpected types.